sm64.ld

```diff
@@ -130,7 +130,7 @@ SECTIONS
    {
       KEEP(BUILD_DIR/asm/entry.o(.text));
 
-      BUILD_DIR/src/game*.o(.text);
+      BUILD_DIR/src/boot*.o(.text);
       BUILD_DIR/src/hvqm*.o(.text);
       BUILD_DIR/src/gdb*.o(.text);
```

Comment:
I should have done this a long time ago but can you remove the GDB references?

Reply:
gdb removed